src/services/databaseservice.cpp

```diff
@@ -243,6 +243,40 @@ bool DatabaseService::setupNoteFolderTables() {
         version = 11;
     }
 
+    if (version < 12) {
+        // create new tag table
+        queryDisk.exec("ALTER TABLE tag RENAME TO _tag");
+        queryDisk.exec("CREATE TABLE IF NOT EXISTS tag ("
```

Comment:
Because sqlite does not support to add columns with non-zero default :( And before filling it is forgotten later, I wanted it to be automatically set.